src/docs/deployment/android.md

```diff
@@ -88,9 +88,10 @@ do not check it into public source control!**
   'Java binary at:'. Then use that fully qualified path
   replacing `java` (at the end) with `keytool`.
   If your path includes space-separated names,
-  such as `Program Files`, use a delimeter appropriate
-  to your platform. For example, use `/"Program Files"/`
-  on Mac/Linux, or `/Program\ Files/` on Windows.
+  such as `Program Files`, use platform-appropriate
+  notation for the names. For example, on Mac/Linux
+  use `/"Program Files"/`, and on Windows use
```

Comment:
Text looks good, but I just noticed that this doesn't render correctly (at least in https://flutter.dev/docs/deployment/android). Should it be these instead?

`"Program Files"`
`Program\ Files`

(And actually, I think the first one might work for both. Maybe we should get a DPE to advise?)

Reply:
@kwalrath, I've staged the site and it looks fine to me. Please confirm:

https://sz-flutter-2.firebaseapp.com/docs/deployment/android

Reply:
The formatting does look better than before (not cut off any more), but are the `\` characters really supposed to be in the strings? And why would there be a `\` _after_ the `"`? This is just a command-line string, not a regex.

![image](https://user-images.githubusercontent.com/2164483/79480443-ecb6b000-7fc2-11ea-9bb2-5aad1ca6826c.png)

Reply:
I think shells on macOS and Linux support escaping using the space character (`cd Program\ Files/`) and the Windows command prompt needs the path is wrapped quotation marks (`cd "C:\Program Files"`)

Reply:
The `\` characters shouldn't be in the examples, right?

Reply:
Also, iirc quotation marks work for Mac/Linux as well as Windows. (Though we might or might not want to say that.)

Reply:
Yeah I would change this to "on Mac/Linux use `Program\ Files` and on Windows use `"Program Files"`